mass bot 2.0.py

```diff
@@ -10,7 +12,8 @@ def rotate(center, radius, seconds, points):
     for i in range(points):
         gui.moveTo(*coords(center, radius, i * unit_angle), seconds/points)
 
+gui.click(x=1081, y=397)

```

Comment:
You used fixed co ordinates. Change and resubmit

Reply:
what do you mean

Reply:
I mean you used `x=1081, y=397`. This wont on others' systems